### PR TITLE
Adding EODIE

### PR DESCRIPTION
**Pull request by samumantha**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by samumantha** [comment]
Noticed I still need to update a few things. -> draft!

File: recipes/eodie/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+source:
+  url : https://gitlab.com/eetun-tiimi/EODIE/-/archive/v{{ version }}/EODIE-v{{ version }}.tar.gz
+  sha256: 09114493834794bb8836e6f1ccc5ff2bd2e8aed995a6cfbc8a68dffebbf32e73
+
+build:
+  skip: true  # [win]
+  number: 0
+
+requirements:
+  host:
```
Comment (bgruening) [inline recipes/eodie/meta.yaml:17]: you can remove everything here, besides python and pip

File: recipes/eodie/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    - fiona
+    - numpy
+    - gdal
+    - pyyaml
+    - matplotlib
+
+test:
+  imports: 
+    - eodie   
+  commands:
```
Comment (bgruening) [inline recipes/eodie/meta.yaml:43]: You could add `python -c 'from osgeo import gdal'` here as well

File: recipes/eodie/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url : https://gitlab.com/eetun-tiimi/EODIE/-/archive/v{{ version }}/EODIE-v{{ version }}.tar.gz
+  sha256: 09114493834794bb8836e6f1ccc5ff2bd2e8aed995a6cfbc8a68dffebbf32e73
+
+build:
+  skip: true  # [win]
```
Comment (bgruening) [inline recipes/eodie/meta.yaml:13]: Why do you skip Windows? Its a Python package and it should work on Windows as well?

**Review by bgruening — COMMENTED** [review]
Thanks @samumantha! A few questions inline.

File: recipes/eodie/meta.yaml (hunk 3 above)
Comment (samumantha) [reply recipes/eodie/meta.yaml]: There used to be some issues with gdal. So did not test it for long. Now it seems fine, just needed to update paths used in tests

**Comment by samumantha** [comment]
@bgruening : Any idea why only windows build fails here? 
I do not quite understand where the run_test.py comes from and then why import eodie fails there (and not in linux/mac build)

**Comment by dopplershift** [comment]
Windows isn't actually building a package because there's a `build.sh` (which only works on mac/linux), but no `bld.bat` file to build on Windows.

**Comment by bgruening** [comment]
Thanks @samumantha

**Pull request merged**